package.json

```diff
@@ -1,34 +1,69 @@
 {
-  "name": "@hashgraph/hedera-wallet-connect",
-  "version": "0.0.2",
+  "name": "walletconnect-hedera",
+  "version": "1.0.0-rc-2",
   "description": "A library to facilitate integrating Hedera with WalletConnect",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/hashgraph/hedera-wallet-connect.git"
+    "url": "git+https://github.com/hgraph-io/hedera-walletconnect-utils.git"
   },
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
-  "scripts": {
-    "build": "tsc",
-    "test": "jest",
-    "prepublishOnly": "npm run build"
+  "main": "./dist/node-cjs.js",
+  "module": "./dist/node-esm.js",
+  "browser": {
+    "./dist/node-cjs.js": "./dist/browser-cjs.js",
+    "./dist/node-esm.js": "./dist/browser-esm.js"
   },
+  "types": "./dist/types/index.d.ts",
+  "author": "Hgraph <support@hgraph.io>",
   "keywords": [
     "hedera",
     "walletconnect",
     "hashgraph"
   ],
-  "license": "Apache-2.0",
+  "license": "MIT",
```

Comment:
The Hedera standard license for all OSS projects is `Apache 2.0`. Please revert this back to an `Apache 2.0` license.